src/main/java/org/jabref/logic/importer/fetcher/DoiFetcher.java

```diff
@@ -89,6 +90,7 @@ public String getName() {
     }
 
     private void doPostCleanup(BibEntry entry) {
+        new PageFieldCleanup().cleanup(entry);
```

Comment:
This is not a good idea to put it here, because it would be called for every type of DOI, and the APS are only a very specific subset. 
I would suggest adding it as a CleanupPreset Step:
https://github.com/JabRef/jabref/blob/1b35f8cb0040fdfb515974e78532598f07e11af2/src/main/java/org/jabref/logic/cleanup/CleanupPreset.java

and also adding it to the Cleanup Dialog maybe as Move article id to pages? (APS).

Reply:
I do think it's the right place, the DOI fetcher is not returning the right/full information, so we improve this here. Of course, you are right and the extraction should only be applied for DOI's from APS (see my comment above).